docs/README.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started
+
+# Pre-requisites
+
+* [Python 3.6.0](https://www.python.org/downloads/release/python-360/)+ (including `pip`)
+* [Docker](https://www.docker.com/) (only needed for `DockerAgent`)
+* [virtualenv](https://virtualenv.pypa.io/en/stable/) (optional, for isolated Python environment)
+
+# Installation
+
+* **OPTIONAL**: Setup an isolated virtual Python environment by running the following commands
+```
+$ virtualenv </path/to/venv>
```

Comment:
I would be absolute with this. You are below in the sourcing of the env. I feel this would make it easier (copy and pasting). If someone knows how to or wants to move the env to a different location then they can.

Reply:
I guess, it makes sense. Look for for the new commit! Thanks!